src/web_interface/rest/rest_missing_analyses.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, List, Set
+
+from flask_restful import Resource
+
+from helperFunctions.database import ConnectTo
+from helperFunctions.rest import success_message
+from storage.db_interface_frontend import FrontEndDbInterface
+
+
+class RestMissingAnalyses(Resource):
+    URL = '/rest/missing_analyses'
+
+    def __init__(self, **kwargs):
+        self.config = kwargs.get('config', None)
+
+    def get(self):
+        with ConnectTo(FrontEndDbInterface, self.config) as db:
+            missing_files = self._make_json_serializable(db.find_missing_files())
+            missing_analyses = self._make_json_serializable(db.find_missing_analyses())
+        return success_message(dict(missing_files=missing_files, missing_analyses=missing_analyses), self.URL)
+
+    @staticmethod
+    def _make_json_serializable(set_dict: Dict[str, Set[str]]) -> Dict[str, List[str]]:
```

Comment:
I'd like to get rid of this utilizing `common_helper_encoding.ReportEncoder` but that's an issue for another PR.